modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PostalAddressResourceTest.java

```diff
@@ -136,6 +156,26 @@ protected Long testGetAccountPostalAddressesPage_getAccountId()
 		return new HashMap<>();
 	}
 
+	@Override
+	protected PostalAddress
+			testGetOrganizationByExternalReferenceCodePostalAddressesPage_addPostalAddress(
+				String externalReferenceCode, PostalAddress postalAddress)
+		throws Exception {
+
+		return _addPostalAddress(
+			postalAddress, _organization.getModelClassName(),
```

Comment:
please use Organization.class.getName() and check if we need to update also the byId method